build-one.sh

```diff
@@ -13,8 +13,18 @@ REPO="khronosgroup/docker-images"
 (
     cd $(dirname $0)
     DOCKERFILE=$1
-    shift
-    TAG=${DOCKERFILE#Dockerfile.}
-    TAG=${TAG%.dockerfile}
-    docker build "$@" . -f "$DOCKERFILE" -t "$REPO:$TAG"
+    DATE=$2
+    shift 2
+    if [ "$1" == "push" ]; then
```

Comment:
If I'm reading this right, the script can either be invoked as

build-one.sh dockerfile-name

or

build-one.sh push dockerfile-name

? If so, would be good to update the comment at lines 5-7 above.

Reply:
Updated. Because of the `shift` it's the third argument.